core/src/main/java/org/infinispan/commands/CreateCacheCommand.java

```diff
@@ -4,6 +4,7 @@
 import java.io.ObjectInput;
 import java.io.ObjectOutput;
 import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
```

Comment:
These changes are needed; see that the TimeoutException caught here is a different class.